.github/workflows/ci.yaml

```diff
@@ -509,7 +462,7 @@ jobs:
         - ${{ github.workspace }}/../../cache/sbt:/root/.sbt
         - ${{ github.workspace }}/../../cache/ivy:/root/.ivy2/cache
         - ${{ github.workspace }}/../../cache/general:/root/.cache
-    needs: [test_non_bootstrapped, test, mima, community_build_a, community_build_b, community_build_c, community_build_forward_compat, test_sbt, test_java8]
```

Comment:
Why was `mima` removed?

Reply:
`git revert` shenanigans. I didn't touch this line.